isotovideo

```diff
@@ -104,10 +109,18 @@ sub version () {
     exit 0;
 }
 
-GetOptions(\%options, 'debug|d', 'workdir=s', 'help|h|?', 'version|v') or usage(1);
+GetOptions(\%options, 'debug|d', 'workdir=s', 'color=s', 'help|h|?', 'version|v') or usage(1);
 usage(0) if $options{help};
 version() if $options{version};
 
+my $color = $options{color} // 'auto';
+# Default to no color if there is no terminal
+$color = 'no' if $color eq 'auto' and (not -t STDOUT or not -t STDERR);
+# User setting has preference, see https://no-color.org/
+delete $ENV{NO_COLOR} if $color eq 'yes';
```

Comment:
Note to self, os-autoinst/openQA/pull/4771 now checks the interface version to pass `--color=no`, so this branch should bump it